.github/workflows/jre11-release.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and push the Docker image for JRE 11
```

Comment:
This file is copied and revised from https://github.com/scalar-labs/docker/blob/main/.github/workflows/jdk8-release.yml